server/docker/update-env.sh

```diff
@@ -4,19 +4,34 @@
 # This script is called by gradle and get the current project version as an input param
 
 if [ $# -lt 1 ]; then
-  echo "USAGE: $0 <VERSION> <DEBUG>"
+  echo "USAGE: $0 <VERSION> <DEBUG> <SMOKE_TEST>"
   exit 1
 fi
 
-echo "VERSION=$1" > .env
+project_version=$1
+# in cases where no params are supplied for debug or smoke tests, later we still need to log something
+is_debug=$([ "$2" = "true" ] && echo "true" || echo "false")
+is_smoke_test=$([ "$3" = "true" ] && echo "true" || echo "false")
```

Comment:
Can we set `is_debug` and `is_smoke_test` to boolean values rather than strings?

Reply:
done